index.bs

```diff
@@ -2310,6 +2317,12 @@ is exposed on the {{Navigator}} object.
 
 The <a>default allowlist</a> for this feature is <code>["self"]</code>.
 
+This specification defines a second <a>policy-controlled feature</a>, identified
+by the token <code>"usb-unrestricted"</code>, that controls whether blocklisted
+USB devices and device interfaces with protected classes can be accessed.
+
+The <a>default allowlist</a> for this feature is <code>["self"]</code>.
```

Comment:
This should say something about how this policy-controlled feature can only be requested by an Isolated Web App. Saying the default allowlist is `["self"]` implies that top-level documents will have it by default, while in reality only IWAs can have it and because of high-watermark permissions they can only have it if it is listed in the manifest. Effectively this means that the default allowlist is `["none"]` but that isn't a value that's understood by the Permissions Policy spec.

Reply:
PTAL

What should I link to for "Isolated Web Apps"? [Explainer?](https://github.com/WICG/isolated-web-apps/blob/main/README.md)

Reply:
Yes, link to the explainer.